### DISCUSSION TITLE
DRES-CMIP-BB4CMIP7-1-0: biomass burning emissions

### DISCUSSION THREAD

**Discussion by armanpouyaei** [PCMDI/input4MIPs_CVs#138] [General]
**Comparison of Black Carbon (BC) Biomass Burning Emissions: CMIP6Plus v20241014, CMIP6 v20161231, and GFED5_Beta**

- **CMIP6Plus v20241014** covers the period from January 1750 to December 2022.
- The current version of **CMIP6Plus** shows emission values similar to those of **CMIP6** and serves as an extended version of **CMIP6 v20161231**, likely using GFED4.1s to extend the dataset beyond December 2015.
- Other variables, such as CO, NOx, and OC, follow similar trends and global distribution patterns across these datasets.
- As expected, **GFED5_Beta** emissions for BC are higher. We hope that the data providers could integrate GFED5 emissions in future versions of **CMIP6Plus**.
- The file formatting of **CMIP6Plus** adheres to **CMIP6** standards.

**Request**: A documentation detailing the changes in each version and outlining future plans from the BB Forcing group would be highly beneficial.

![output1](https://github.com/user-attachments/assets/fe825813-1386-4c8e-a3ee-7eb7d6552c38)
![output2](https://github.com/user-attachments/assets/4c1a3305-262f-412a-9159-f804e5a3ff85)

**Comment by znicholls** [comment]
Thanks @armanpouyaei, very clear! One clarification

> * Future versions of **CMIP6Plus** are expected to integrate GFED5 emissions

Is this true? It would be good to check with the data providers directly as my impression was that GFED5 would not be ready in time for CMIP6Plus/CMIP7.

**Reply by armanpouyaei** [reply]
You're right @znicholls! We hope that at some point in the future, the data providers will be able to integrate GFED5 into this dataset, though of course, it would be a very challenging and time-consuming process. I'll edit my post to reflect this.

**Comment by durack1** [comment]
@mjevanmarle ping!

**Comment by chrisroadmap** [comment]
Hi everyone, I spotted an issue when using `esgpull` with the CMIP7 biomass emissions, which is that I can't find an obvious way to obtain the NMVOC bulk variable.

If I run

```
$ esgpull search --all project:input4MIPs mip_era:CMIP6Plus source_id:DRES-CMIP-BB4CMIP7-1-0 grid_label:gn
```

then I get the expected dataset listed

```
 id  │                                                 dataset                                                  │ # │   size    
═════╪══════════════════════════════════════════════════════════════════════════════════════════════════════════╪═══╪═══════════
  ... 
  99 │ input4MIPs.CMIP6Plus.CMIP.DRES.DRES-CMIP-BB4CMIP7-1-0.atmos.mon.NMVOC_bulk.gn.v20241014                  │ 2 │   1.2 GiB
  ...
```

however, I can't find a combination of facet names that will return the dataset (`variable:NMVOC`, `variable:NMVOC_bulk`, `variable:NMVOC-bulk` don't work).

The dataset exists on the ESGF web tool: https://aims2.llnl.gov/search?project=input4MIPs&activeFacets=%7B%22mip_era%22%3A%22CMIP6Plus%22%2C%22source_id%22%3A%22DRES-CMIP-BB4CMIP7-1-0%22%2C%22variable_id%22%3A%22Multiple%22%7D under the variable `Multiple`, returning 223 results.

The files exist:
- https://esgf-data2.llnl.gov/thredds/dodsC/user_pub_work/input4MIPs/CMIP6Plus/CMIP/DRES/DRES-CMIP-BB4CMIP7-1-0/atmos/mon/NMVOC_bulk/gn/v20241014/NMVOC_bulk_input4MIPs_emissions_CMIP_DRES-CMIP-BB4CMIP7-1-0_gn_175001-189912.nc
- https://esgf-data2.llnl.gov/thredds/dodsC/user_pub_work/input4MIPs/CMIP6Plus/CMIP/DRES/DRES-CMIP-BB4CMIP7-1-0/atmos/mon/NMVOC_bulk/gn/v20241014/NMVOC_bulk_input4MIPs_emissions_CMIP_DRES-CMIP-BB4CMIP7-1-0_gn_190001-202212.nc

I think the `Multiple` variable needs renaming in these datasets.

**Reply by znichollscr** [reply]
Nice pick up @chrisroadmap. I think part of the issue is that the variable name is "NMVOC_bulk" in the filename, which means the DRS doesn't parse properly (if we you have an underscore in the variable part of the filename, everything breaks). Good one to fix for the next round, which should then flow through to this appearing properly on ESGF.

**Reply by durack1** [reply]
Agreed, @chrisroadmap we've been trying to catch these issues BEFORE data are published, but some have slipped through, which makes the data download/access a little trickier than it should be. As @znichollscr noted, we've got this identified for a "fix", see https://github.com/PCMDI/input4MIPs_CVs/issues/133 - feel free to add other issues if they can be identified

**Comment by edblanchard** [comment]
Hello everyone

We recently published a paper showing that observed boreal biomass burning emissions (BBEs) since 2015 have been much higher in observations (GFED4s) than those in CMIP6 scenarios. By re-running CESM2 with ramped up boreal BBEs, based on observed values and trends, over 2015-2060, we show these enhanced emissions have significant impacts on climate trends, slowing down global warming (by 12%) and Arctic warming (by 38%), reducing Arctic sea ice loss, and shifting tropical precipitation southward. We hope this helps inform the design of future boreal BBEs in CMIP7. 

Paper is [here](https://www.pnas.org/doi/abs/10.1073/pnas.2424614122)

Best,
Ed

<img width="618" alt="Screenshot 2025-06-13 at 4 50 56 PM" src="https://github.com/user-attachments/assets/b4481f3e-51da-47e2-be32-51e5f5c606a1" />
